crowdsec-docs/unversioned/user_guides/alert_context.md

```diff
@@ -4,41 +4,30 @@ title: Contextualize Alerts
 sidebar_position: 10
 ---
 
-# Alert Contextualization
+## Introduction
 
-While CrowdSec doesn't store any logs after processing it, it can be useful to get some context about why an alert has been triggered (source machine, targeted FQDN ...).
+CrowdSec doesn't store any logs after processing them, but you can join information to an alert with what we call *alert context*:
```

Comment:
CrowdSec doesn't store raw log information once it has finished processing them. However, you can send additional meta data within an alert with a feature we call *alert context*

Reply:
Maybe to not confused `meta` data with our `Meta` key we can just say `additional data within an alert with`